var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -46,6 +47,17 @@ class Protobuf(CMakePackage):
     # first fixed in 3.4.0: https://github.com/google/protobuf/pull/3406
     patch('pkgconfig.patch', when='@:3.3.2')
 
+    def fetch_remote_versions(self):
+        """fix for https://github.com/LLNL/spack/issues/5356"""
```

Comment:
IMO a description of the issue would be good here vs. a link.

Reply:
all right, will do!
the idea of the link was just: in case we improve upon the situation generally, the link can be grep-ed and updated.